install-storageos/010-install-etcd.md

```diff
@@ -0,0 +1,23 @@
+StorageOS uses etcd to store its metadata. We will install etcd on node01 as systemd service. Check
+https://docs.storageos.com/docs/prerequisites/etcd/ if you want a
+production topology. The following method is based on the manual installation in the docs that's been automated with a script.
+
+First, you need to add the ip of the ETCD host in the script file with this command.
+
+`echo export ETCD_HOST=${ETCD_HOST} | cat - install-etcd.sh > temp && mv temp install-etcd.sh`{{execute}}
+
+Then, you need to send the script to node01 to be able to execute it there.
+
+`scp install-etcd.sh node01:~/install-etcd.sh`{{execute}}
+
+Then, you need to ssh into the node to execute the installation script.
+
+`ssh node01`{{execute}}
```

Comment:
Why not combine these into one command? ssh node01 'bash -c ./install-etcd' or whatever.